src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -215,6 +215,11 @@ std::string toCodeString(double val) {
     return buf;
 }
 
+std::string toShapeInfoString(size_t idx) {
+    snprintf(buf, sizeof(buf), "shape_info[%zu]", idx);
+    return buf;
+}
```

Comment:
I am curious whether this provides much benefit. shape_info[idx] is shorter than using toShapeInfoString[idx]. I think to make such a new handle more meaningful, we may need more functionality something like, ..
```

std::string toShapeInfoString(arg_idx, data_idx, bool is_output) {
    size_t actual_idx;
    actual_idx = num_of_inputs * 6 * (is_output ? 1 : 0) + 6 * arg_idx + data_idx;
    sprintf(buf, sizeof(buf), "shape_info[%zu]", actual_idx);
}

```

Reply:
Added

Reply:
Do we still need original toShapeInfoString(size_t idx) ?

Reply:
deleted